hooks/post-commit.py

```diff
@@ -5,7 +5,7 @@
 
 
 def run_command(command):
-    return subprocess.check_output(command.split(), shell=True).rstrip()
+    return subprocess.check_output(command.split()).rstrip()
```

Comment:
Sorry for not comment it. When I run this line with shell=True I get the following error:

Traceback (most recent call last):
  File "/Users/adaofeliz/bin/post-commit.py", line 11, in <module>
    tld = run_command('git rev-parse --show-toplevel')
  File "/Users/adaofeliz/bin/post-commit.py", line 8, in run_command
    return subprocess.check_output(command.split(), shell=True).rstrip()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 573, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
subprocess.CalledProcessError: Command '['git', 'rev-parse', '--show-toplevel']' returned non-zero exit status 1

By removing it, this errors does not occur.